notebooks/OpenMLxProbabl-hackathon.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "---\n",
+    "title: OpenML x Probabl Hackathon\n",
+    "topic: hacakathon\n",
+    "author: Subhaditya Mukherjee , Emily Chen\n",
+    "date: 09-19-2024\n",
+    "format:\n",
+    "  html:\n",
+    "    code-fold: false\n",
+    "--"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "[scikit-learn](http://scikit-learn.org) is a free and open-source machine learning library for the Python programming language, while [OpenML](http://openml.org) is an open platform for sharing datasets, algorithms, and experiments. While our teams have been working together for many years, we do not always have the time to meet in person. When we do get the chance though, many interesting discussions stem from those conversations.\n",
+    "\n",
+    "We recently had a developers hackathon at the Paris office of [Probabl](https://probabl.ai/about), the official operating brand of scikit-learn, focused on maintaining and expanding open-source ML in Europe and beyond). We met to discuss not only the state of AI and how both our organizations fit in, but also to brainstorm solutions to challenges faced by our developers and communities.  \n",
+    "\n",
+    "<div style=\"text-align:center\">\n",
+    "<img alt=\"View from the office\" src=\"../images/blogs/openmlxsci/openmlxsci-1.webp\" style=\"width:40%\">\n",
+    "</div>\n",
+    "\n",
+    "Many interesting topics were brought up, most of which were not only relevant to us but also to the broader open-source community. In the spirit of open-source, we wanted to share these insights with you."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information about this hackathon or just to chat with us, feel free to reach out to us on [OpenML Email](mailto:openml-labs@tue.nl), [Probabl](https://probabl.ai/about). To contact the authors, send them an email here - [Subhaditya Mukherjee](mailto:msubhaditya@gmail.com), [Emily Chen](mailto:emilyxinyi.chen2021@gmail.com)."
+   ]
+  },
```

Comment:
Would put this cell at the end of the blogpost.

Reply:
I assume you mean the reaching out cell. I will do that in the next commit then.